utils/line-directive

```diff
@@ -262,13 +271,19 @@ def run():
     glitch in [fox.box:28 assert six
     EOF
     >>> print(subprocess.check_output([sys.executable, __file__, 'foo.bar',
-    ...                                '21', target1.name]).rstrip())
+    ...                                '21', target1.name],
+    ...                               universal_newlines=True))
     5
+    <BLANKLINE>
```

Comment:
@dabrahams I'm really not sure what's causing these to be necessary now that we've removed `rstrip` but the tests (on Wiindows, not sure about other platforms) fail without this.